src/v2/Apps/Artist/routes.tsx

```diff
@@ -222,6 +225,12 @@ export const routes: RouteConfig[] = [
             }
           )
 
+          const user = props.context.user
+          filterParams.shouldFetchArtistSeriesData = userHasLabFeature(
```

Comment:
This is the output of `prepareVariables`, so it should match the shape of the GraphQL variables. For the artwork page (no filter query params), that's in https://github.com/artsy/force/blob/4301e22db2ffe9c5da0859c46e967621f65c69b4/src/v2/Apps/Artwork/routes.tsx#L21